packages/server/package.json

```diff
@@ -29,6 +29,7 @@
   "module": "dist/server.esm.js",
   "types": "dist/packages/server/src/index.d.ts",
   "dependencies": {
+    "@babel/core": "7.11.6",
```

Comment:
Let's use a `^` range here so that we use the same version Next.js. Or, even we could maybe remove it from here since next includes it.